sbt-web-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/WebScalaJSBundlerPlugin.scala

```diff
@@ -70,8 +84,8 @@ object WebScalaJSBundlerPlugin extends AutoPlugin {
             .value
 
         val bundlesWithSourceMaps =
-          bundles.flatMap { case ((file, path), sourceMapsEnabled) =>
-            if (sourceMapsEnabled) {
+          bundles.flatMap { case ((file, path), bundleSourceMapsEnabled) =>
+            if (bundleSourceMapsEnabled && !file.name.endsWith("loader.js")) {
```

Comment:
Can you explain the changes made to this file? Why were they necessary?

Reply:
This skips looking for a source map file top copy along with loader.js, since it hasn't got one.

Reply:
I've removed the magic value for -loader.

Reply:
The other change forces a rebuild when js assets in src/main/resources are changed. It's probably relevant outside the context of this PR but I noticed the regression when tasking all the change detection paths I could think of.